### PR TITLE
[Keyboard] Add Kimiko keyboard 

### PR DESCRIPTION
**Pull request by BenRoe**
Add the Kimiko Split Keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/keycapsss/kimiko/readme.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Hardware Supported: Pro Micro 5V/16Mhz and compatible  
+Hardware Availability: [keycapsss.com](https://keycapsss.com)
+
+Make firmware .hex for this keyboard (after setting up your build environment):
+
+    make keycapsss/kimiko:default
+
+Example of flashing this keyboard (or use [QMK Toolbox](https://github.com/qmk/qmk_toolbox)):
+
+    make keycapsss/kimiko:default:dfu
```
Comment (zvecr) [inline keyboards/keycapsss/kimiko/readme.md:21]:
```suggestion
    make keycapsss/kimiko:default:flash
```

File: keyboards/keycapsss/kimiko/readme.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+A split keyboard with 4x6 vertically staggered keys and thumb keys.
+
+- Per key RGB led's (SK6812 Mini-E)
+- 6 underglow RGB led's per side (SK6812 Mini)
+- Support for 1 rotary encoder per side (two possible positions)
+
+Keyboard Maintainer: [BenRoe](https://github.com/BenRoe/) [@keycapsss](https://twitter.com/keycapsss)  
+Hardware Supported: Pro Micro 5V/16Mhz and compatible  
+Hardware Availability: [keycapsss.com](https://keycapsss.com)
```
Comment (zvecr) [inline keyboards/keycapsss/kimiko/readme.md:13]:
```suggestion
* Keyboard Maintainer: [BenRoe](https://github.com/BenRoe/) [@keycapsss](https://twitter.com/keycapsss)
* Hardware Supported: Pro Micro 5V/16Mhz and compatible
* Hardware Availability: [keycapsss.com](https://keycapsss.com)
```

File: keyboards/keycapsss/kimiko/rules.mk [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (zvecr) [inline keyboards/keycapsss/kimiko/rules.mk:11]:
```suggestion
```

File: keyboards/keycapsss/kimiko/rules.mk [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+EXTRAKEY_ENABLE = yes	# Audio control and System control
+CONSOLE_ENABLE = no		# Console for debug
+COMMAND_ENABLE = no		# Commands for debug and configuration
+NKRO_ENABLE = no		# Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no	# Enable keyboard backlight functionality
+MIDI_ENABLE = no		# MIDI controls
+AUDIO_ENABLE = no		# Audio output on port C6
+UNICODE_ENABLE = no		# Unicode
+BLUETOOTH_ENABLE = no	# Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = no	# Enable WS2812 RGB underlight.
```
Comment (zvecr) [inline keyboards/keycapsss/kimiko/rules.mk:29]:
```suggestion
# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/keycapsss/kimiko/rules.mk [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+SPLIT_KEYBOARD = yes
+
+
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no	# Breathing sleep LED during USB suspend
```
Comment (fauxpark) [inline keyboards/keycapsss/kimiko/rules.mk:34]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/keycapsss/kimiko/readme.md (hunk 1 above)
Comment (BenRoe) [reply keyboards/keycapsss/kimiko/readme.md:21]: @zvecr qmk choose the right programmer depending on the `rules.mk`?

Comment (drashna) [reply keyboards/keycapsss/kimiko/readme.md:21]: Yes. And specifically, based on the bootloader set, and MCU type.

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**